qiskit/algorithms/optimizers/qnspsa.py

```diff
@@ -141,6 +139,9 @@ def __init__(
             callback: A callback function passed information in each iteration step. The
                 information is, in this order: the parameters, the function value, the number
                 of function evaluations, the stepsize, whether the step was accepted.
+            termination_callback: A callback function executed at the end of each iteration step. The
+                arguments are, in this order: current parameters, estimate of the objective
+                If the callback returns True, the optimization is aborted
```

Comment:
Instead of using `aborted` which has implications of being ended due to problems or failure, can we use some other word e.g. `terminated`, `ended`, `concluded` - same comment for other places this is used, such as in the logging.